spec/controllers/email_authentications/invitations_controller_spec.rb

```diff
@@ -5,6 +5,7 @@
     @request.env['devise.mapping'] = Devise.mappings[:email_authentication]
     admin = create(:admin, :owner)
     sign_in(admin.email_authentication)
+    ActionMailer::Base.deliveries.clear
```

Comment:
Looks like the spec on line 63 is failing because it expects a delivery to be available 🤔